src/torch/12-deeprank-train/deeprank-train.lua

```diff
@@ -26,7 +27,7 @@ function getSimilarityModel()
   --leftImage:add(nn.SpatialConvolutionMM(512, 126, 1, 1, 1, 1, 0, 0):cuda())
```

Comment:
Not fussy about it, honestly, so will leave it up to you. In general, I think people would not look at that sort of thing for reference, so I usually clean it up. If you think it may be beneficial for this project - I've never worked on it, so hard for me to tell - then it's more than fine to leave it there.